cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -680,6 +681,90 @@ def verify_name(source_usage_key, parent_usage_key, expected_name, display_name=
         verify_name(self.seq_usage_key, self.chapter_usage_key, "customized name", display_name="customized name")
 
 
+@ddt.ddt
+class TestMoveItem(ItemTest):
```

Comment:
I think this should go in ``TestCourseOutline`` which already exists and which covers a lot of the kinds of tests that are needed. I think the course outline should take a ``concise=true`` parameter or something to give a more performant response for the move modal.